config.go

```diff
@@ -60,6 +59,10 @@ const (
 	hyperstartAgent = "hyperstart"
 )
 
+// XXX: set by build
+var libExecDir = ""
```

Comment:
libexecDir maybe

I think it'd be nice to have sane defaults in this variable so we don't have to check for "" everywhere. NUL value not allowed! :)